### PR TITLE
[trends] don't allow persons on formula

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- addresses #6797 by not allowing persons on formula for now
- conceptually, showing persons based on formula transformed numbers doesnt seem intuitive
*If this affects the frontend, include screenshots.*  

## How did you test this code?

*Please describe.*
manual qa

### CODE REVIEW

File: frontend/src/scenes/trends/viz/ActionsLineGraph.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export function ActionsLineGraph({
             showPersonsModal={showPersonsModal}
             tooltipPreferAltTitle={filters.insight === ViewType.STICKINESS}
             onClick={
-                dashboardItemId
+                dashboardItemId || filters.formula
```
Comment (hazzadous) [inline frontend/src/scenes/trends/viz/ActionsLineGraph.tsx:42]: It's not obvious to me why these cases should mean that we don't have `onClick`. Can we add either a comment or provide as a function `isPersonsListValidForFilter`.

**Review by hazzadous — APPROVED** [review]
Looks ok. I'm not sure how we can automated test this easily?

There seems to be a pattern in a lot of these which is:

 1. retrieve some breakdown of events
 2. display this breakdown
 3. have click through to detail

What do you think of, rather than having the frontend need to understand the nuance of the results it's getting from the backend, we instead in the response clearly define if and how you can get persons for a specific value/breakdown? The might not work in all cases I guess, so perhaps good not to abstract this too early, but we can try with e.g. correlations? Or maybe there's another we could try with.

i.e. keep coupled logic close together code wise, and remove coupling with the frontend, where perhaps we don't want it.

File: frontend/src/scenes/trends/viz/ActionsLineGraph.tsx (hunk 1 above)
Comment (EDsCODE) [reply frontend/src/scenes/trends/viz/ActionsLineGraph.tsx:42]: good point

**Comment by EDsCODE** [comment]
> What do you think of, rather than having the frontend need to understand the nuance of the results it's getting from the backend, we instead in the response clearly define if and how you can get persons for a specific value/breakdown? 

I like this. What about doing this more in depth? When you retrieve insight data, we can provide a follow on link that will return persons for that data if you provide x, y, and z parameters. This can reduce a lot of the random if statements that we need to control for each specific case

**Pull request merged**